pkgs/tools/misc/scriptcommunicator/default.nix

```diff
@@ -0,0 +1,93 @@
+{ stdenv
+, fetchFromGitHub, qmake, pkgconfig
+, libpng, zlib, unzip
+, qt5, qttools, qtserialport, qtscript, qtmultimedia
+, makeWrapper, makeDesktopItem
+}:
+
+let
+  pname = "scriptcommunicator";
+  version = "05.10";
+
+  desktopItem = makeDesktopItem {
```

Comment:
Nice! Good job, thinking about that.
I think `desktopName` should be `ScriptCommunicator`,
`GenericName` should be something like "Serial Terminal" or "Data Terminal" (I like the former better),
you can set `name = pname;`
I think exec should be the absolute path but I'm not sure if `$out/bin/ScriptCommunicator` works here.